packages/testing/config/storybook/main.js

```diff
@@ -120,7 +122,13 @@ const baseConfig = {
     ]
 
     // ** LOADERS **
-    sbConfig.module.rules = rwConfig.module.rules
+    const sbMdxRule = sbConfig.module.rules.find(
+      (rule) => rule.test.toString() === /(stories|story)\.mdx$/.toString()
+    )
+    console.assert(sbMdxRule, 'Storybook MDX rule not found')
```

Comment:
Do we need this assert here - not sure I understand why we would need it? If the SB team decided to change the regex, might start causing problems, no?

Reply:
Also... is it a bad idea to pull in all the default config?